projects/rocprofiler-sdk/source/include/rocprofiler-sdk/rccl/details/api_trace.h

```diff
@@ -223,7 +223,14 @@ typedef ncclResult_t (*ncclCommRegister_fn_t)(const ncclComm_t comm,
                                               void**           handle);
 
 typedef ncclResult_t (*ncclCommDeregister_fn_t)(const ncclComm_t comm, void* handle);
-
+typedef ncclResult_t (*ncclAllReduceWithBias_fn_t)(const void*      sendbuff,
+                                                   void*            recvbuff,
+                                                   size_t           count,
+                                                   ncclDataType_t   datatype,
+                                                   ncclRedOp_t      op,
+                                                   struct ncclComm* comm,
+                                                   hipStream_t      stream,
+                                                   const void*      acc);
```

Comment:
In the RCCL repo, [this function was not added after `ncclCommDeregister_fn_t`](https://github.com/ROCm/rccl/blob/277747c19984ce54a02741cf1a3334ec5f3d4987/src/include/api_trace.h#L64-L67), so I added the new on here, after the last declaration for readability. 

Should I keep this in sync with RCCL's copy instead?

Reply:
Done, thanks! 
(your review was dismissed because of the update)